source/features/esc-to-deselect-line.tsx

```diff
@@ -15,7 +15,7 @@ function isLineSelected(): boolean {
 function listener({key, target}: KeyboardEvent): void {
 	if (key === 'Escape' && isLineSelected() && !isEditable(target)) {
 		location.hash = '#no-line'; // Update UI, without `scroll-to-top` behavior
-		history.replaceState({}, document.title, location.pathname); // Drop remaining # from url
+		history.replaceState(null, document.title, location.pathname); // Drop remaining # from url
```

Comment:
🤔 this shouldn't pass lint
It should be undefined

Reply:
The spec says the type can be `any` so `undefined` is OK.